test/list.v1.test.ts

```diff
@@ -12,11 +12,14 @@ describe('List endpoints for v1.1 API', () => {
 
   it('.createList/.updateList/.listOwnerships/.removeList/.list - Create, update, get and delete a list', async () => {
     const newList = await client.v1.createList({ name: 'cats', mode: 'private' });
+
+    await sleepTest(1000);
```

Comment:
it doesn't seem to solve the issue 😪

Reply:
It seems that Twitter returns a 404 for deleteList but... it deletes the list anyways 😓